src/Adapter/Carrier/CommandHandler/AddModuleCarrierHandler.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Adapter\Carrier\CommandHandler;
+
+use Carrier;
+use PrestaShop\PrestaShop\Core\Domain\Carrier\Command\AddModuleCarrierCommand;
+use PrestaShop\PrestaShop\Core\Domain\Carrier\CommandHandler\AddModuleCarrierHandlerInterface;
+use PrestaShop\PrestaShop\Core\Domain\Carrier\Exception\CarrierException;
+use PrestaShop\PrestaShop\Core\Domain\Carrier\ValueObject\CarrierId;
+use PrestaShopException;
+
+/**
+ * Handles AddModuleCarrierCommand using legacy object model
+ */
+final class AddModuleCarrierHandler extends AbstractAddCarrierHandler implements AddModuleCarrierHandlerInterface
```

Comment:
I don't. But I guess there will be no use cases unless i create carriers module.. I'm afraid it would be forgotten to migrate at all.
You mean to get rid of AddModuleCarrierCommand and instead use something like $isModule property to preset all redundant data based on it?